### ISSUE TITLE
Question to JQueryUI Globalize.js and cldrjs

### ISSUE THREAD

**Issue by Patrick1701** [sebfz1/wicket-jquery-ui#205]
Hi Sebastien,
using ```JQueryGlobalizeResourceReference``` I can add the globalize.js to a page.

Does this solution follow the idea of having a plain-javascript on page like this example?
(see https://github.com/jquery/globalize/tree/master/examples/plain-javascript)

Do I have to add the cldrjs.js files and CLDR JSON data manually?

kind regards
Patrick

**Comment by sebfz1** [comment]
Hi Patrick,

I'm afraid I'm using Globalize v0.x (can't find out the version in .js file, but quite sure), so it is not designed to work with CLDR. I should upgrade to v1.x...

Are you looking for something that is not covered by kendo culture?
BTW, I see #193 is still open, can we close it?

**Comment by Patrick1701** [comment]
> Are you looking for something that is not covered by kendo culture?

I'm using the JQueryUI Spinner... yes, there is a KendoUI NumericTextInput equivalent featuring cultures, but we don't like it. Doesn't fit into our ui, well.

We need to enter fractional digits and by using the jqueryui spinner, and I need to have ```,``` instead of ```.``` and I think jquery-ui globalize.js will handle this for me. Thats why I'm currently doing some research.

___

> BTW, I see #193 is still open, can we close it?

I'm not sure... I dont think so... did you add message js files like for instance ```kendo.messages.bg-BG.min.js``` and others into the artefact?
(http://docs.telerik.com/kendo-ui/framework/localization/overview)

**Comment by sebfz1** [comment]
Right, I only have `culture` files not `message` files... I will try to add these asap, thanks for the reminder ;)
I will also upgrade Globalize to v1.x, but I'm not sure when... I will let you know!

**Comment by Patrick1701** [comment]
>I will try to add these asap, thanks for the reminder

Also thanx!

>I will also upgrade Globalize to v1.x, but I'm not sure when... I will let you know!

Ok, cool, please have a look how ```cldrjs``` works.
I'm still researching, and I'm still not sure, if ```cldrjs``` will dynamically lazy-load missing i18n file dependencies through the web direct to the browser or not.
If so, I'm not allowed to use it.

If ```globalize 1.0``` with ```cldrjs``` works like this, and you would like to use this feature, to avoid having the maintenance overhead for i18n files (what can I comprehend ;-) ), then I will need a feature to do the maintenance in-house.
But lets see this later on...

**Comment by Patrick1701** [comment]
Hi Sebastien,
any news here? :-)

kind regards
Patrick

**Comment by sebfz1** [comment]
Hi Patrick,

Unfortunately, not at all! :-/

Best Regards,
Sebastien.
